main.rb

```diff
@@ -0,0 +1,152 @@
+module Enumerable
+  def my_each
+    return to_enum unless block_given?
+
+    array = to_a
+    index = 0
+    while index < size
+      yield(array[index])
+      index += 1
+    end
+    array
+  end
```

Comment:
You should have a working myEach method. This should return the same thing as each method in ruby 
![my_each](https://user-images.githubusercontent.com/32361963/95429088-92375480-0952-11eb-9e91-df529432a16c.png).

The problem in this method is that when a block is given when self is a range it should return the range itself after calling the given block once for each element in self. [reference](https://ruby-doc.org/core-2.6.4/Array.html#method-i-each).

Reply:
Please show ur test, because I might not understand what you asked for, because to me I have a working my_each class, so I don't know what you ask from me